service/controller/v1/resource/drainer/create.go

```diff
@@ -34,7 +33,6 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 
 	if drainerConfig.Status.HasFinalCondition() {
 		r.logger.LogCtx(ctx, "level", "debug", "message", "node config status already has final state", "clusterID", clusterID)
-		resourcecanceledcontext.SetCanceled(ctx)
```

Comment:
Just noticed that active cancelation is not necessary here because for normal resources simply returning in a function cancels. This is not a CRUD resource which cancelation is for. Anyways I want to have the logging applied for usual cancelation so the information is there and the pattern aligns with the rest of the codebase.